### PR TITLE
Update hw-event-proxy image in e2e tests

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

**Review by nishant-parekh — APPROVED** [review]

File: core-services/image-mirroring/openshift/mapping_origin_4_10 [hunk 1]
```diff
@@ -128,6 +128,7 @@ registry.ci.openshift.org/origin/4.10:haproxy-router quay.io/openshift/origin-ha
 registry.ci.openshift.org/origin/4.10:hco-oc-bin-image quay.io/openshift/origin-hco-oc-bin-image:4.10 quay.io/openshift/origin-hco-oc-bin-image:4.10.0 quay.io/openshift/origin-hco-oc-bin-image:latest
 registry.ci.openshift.org/origin/4.10:hello-openshift quay.io/openshift/origin-hello-openshift:4.10 quay.io/openshift/origin-hello-openshift:4.10.0 quay.io/openshift/origin-hello-openshift:latest
 registry.ci.openshift.org/origin/4.10:helm-operator quay.io/openshift/origin-helm-operator:4.10 quay.io/openshift/origin-helm-operator:4.10.0 quay.io/openshift/origin-helm-operator:latest
+registry.ci.openshift.org/origin/4.10:hw-event-proxy quay.io/openshift/origin-hw-event-proxy:4.10 quay.io/openshift/origin-hw-event-proxy:4.10.0 quay.io/openshift/origin-hw-event-proxy:latest
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_10:131]: What is a `hw-event-proxy`?  Can you describe a bit more what it does, and what `hw` means?  That'll help me recommend a name.

Comment (jzding) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: It is `Hardware Event Proxy`. It provides low-latency delivery service for Redfish hardware events.
https://github.com/redhat-cne/hw-event-proxy

Comment (smarterclayton) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: What component is it related to?  Ironic?  Bare-metal?  Machine api?

Comment (jzding) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: The event subscription is done at bare-metal https://github.com/metal3-io/baremetal-operator/pull/1014. 
The `Hardware Event Proxy` itself does not have dependencies on other Openshift components. Once it is deployed user can use its URL as destination address to create a  BMCEventSubscription CRD and start to receive Redfish events. The CRD is in openshift-machine-api namespace.
```
apiVersion: metal3.io/v1alpha1
kind: BMCEventSubscription
metadata:
  name: sub-01
  namespace: openshift-machine-api
spec:
  hostRef: cnfdd4-worker-1
  destination: https://hw-event-proxy-cloud-native-events.apps.cnfdd4.t5g.lab.eng.bos.redhat.com/webhook
  context: ""
  ```

Comment (smarterclayton) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: Would anyone ever user this outside of the bare metal context?

Comment (smarterclayton) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: I might suggest using `baremetal-hardware-event-proxy` as the name of the image

Comment (jzding) [reply core-services/image-mirroring/openshift/mapping_origin_4_10:131]: It only applies to bare metal context. I will make the changes.

Comment (jzding) [reply core-services/image-mirroring/openshift/mapping_origin_4_10]: done

**Review by aneeshkp — APPROVED** [review]
/lgtm

**Pull request merged**